packages/@aws-cdk/aws-ecr-assets/lib/image-asset.ts

```diff
@@ -47,7 +47,7 @@ export class NetworkMode {
   /**
    * @param mode The networking mode to use for docker build
    */
-  private constructor(public readonly mode: string) {}
+  private constructor(public readonly mode: string) { }
```

Comment:
Just to confirm: Did these come from the linter? If so - great! Otherwise I'd ask you to keep unrelated formatting changes to a minimum.

Reply:
Yep, this came from the linter.